modules/apps/jaggery/emm/src/emm/modules/device.js

```diff
@@ -319,16 +319,19 @@ deviceModule = function () {
     publicMethods.getLicense = function (deviceType, languageCode) {
         var url;
         var license;
-        if(deviceType != null && deviceType != undefined && deviceType == "windows"){
+        if (deviceType != null && deviceType != undefined && deviceType == "windows") {
             url = mdmProps["httpURL"] + "/mdm-windows-agent/services/device/license";
-        } else{
+        } else if (deviceType != null && deviceType != undefined && deviceType == "ios") {
```

Comment:
you can simply use - if (deviceType == "ios"), and use a constant for accessing deviceType